lib/util.rb

```diff
@@ -56,7 +56,7 @@ def repo_has_branch?(repo_name, branch_name)
 
 def repo_has_pr?(repo_name, branch_name)
   org_name = repo_name.split("/").first
-  Octokit.pull_requests(repo_name, head: "#{org_name}:#{branch_name}").count > 0
+  Octokit.pull_requests(repo_name, head: "#{org_name}:#{branch_name}").count.positive?
```

Comment:
Let's fix up this change to the one where the `repo_has_pr?` method was defined. This change isn't anything to do with `User is notified when they are rate limited`.